api/spec/requests/spree/api/v2/storefront/account_spec.rb

```diff
@@ -91,6 +93,8 @@
         password_confirmation: 'newpassword123',
         bill_address_id: default_bill_address.id.to_s,
         ship_address_id: default_ship_address.id.to_s,
+        first_name: 'Joe',
```

Comment:
@aleksandarpetrushev let's use only fictional people names :)

Reply:
Sure, also need to add docs